client/src/views/workspaces/WorkspaceSharingModal.vue

```diff
@@ -65,35 +105,54 @@
               :is-current-user="true"
               class="current-user"
             />
-
-            <workspace-user-role
+            <div
               v-for="entry in filteredSharedUserRoles"
-              :disabled="isSelectDisabled(entry[1])"
-              :key="entry[0].id"
-              :user="entry[0]"
-              :role="entry[1]"
-              :client-profile="ownProfile"
-              :client-role="ownRole"
-              @role-update="updateUserRole"
-            />
+              :key="`${entry.user.id}-${entry.role}`"
```

Comment:
Is there a reason why you updated the key to add the role? The userId should already be unique.